paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerViewModel.kt

```diff
@@ -36,6 +36,10 @@ internal class FlowControllerViewModel(
     var state: DefaultFlowController.State?
         get() = handle[STATE_KEY]
         set(value) {
+            flowControllerStateComponent.linkHandler.setupLink(
```

Comment:
What if this is restored from process death? Should we do a collect, and side effect there instead?